validator/testcases/javascript/entity_values.py

```diff
@@ -171,6 +171,33 @@ def on_override(wrapper, arguments, traverser):
     return {'return': on_override}
 
 
+@register_entity('nsIPermissionManager.add')
+@register_entity('nsIPermissionManager.addFromPrincipal')
+@register_entity('nsIPermissionManager.remove')
+@register_entity('nsIPermissionManager.removeFromPrincipal')
+@register_entity('nsIPermissionManager.removeAll')
+@register_entity('nsIPermissionManager.testExactPermission')
+@register_entity('nsIPermissionManager.testExactPermissionFromPrincipal')
+@register_entity('nsIPermissionManager.testPermission')
+@register_entity('nsIPermissionManager.testPermissionFromPrincipal')
+def nsIPermissionManager(traverser):
+    traverser.err.warning(
+        err_id=('testcases_javascript_entity_values',
```

Comment:
I think you need to set the version you want the compatibility error to happen for in this call.